Source/VisualRecognitionV3/VisualRecognition.swift

```diff
@@ -41,8 +41,40 @@ public class VisualRecognition {
        in "YYYY-MM-DD" format.
      */
     public init(apiKey: String, version: String) {
-        self.credentials = .apiKey(name: "api_key", key: apiKey, in: .query)
+        self.authMethod = APIKeyAuthentication(name: "api_key", key: apiKey, location: .query)
         self.version = version
+        self.serviceURL = "https://gateway-a.watsonplatform.net/visual-recognition/api"
```

Comment:
Something that Jeff brought up is that we need to ensure that if the user sets the service URL we should not automatically overwrite what they have set. That might be an issue for another time though.

Reply:
That's not an issue for Swift, since the default is set when the service is instantiated, and the only way to supply a custom service URL is by setting the attribute _after_ the service is instantiated.